src/components/shared/Entity/ExistingEntityCards/Store/create.ts

```diff
@@ -61,7 +61,7 @@ const getInitialState = (
                 const { data, error } = await getLiveSpecsByConnectorId(
                     entityType,
                     connectorId,
-                    'ops'
+                    ['ops', 'demo']
```

Comment:
I am wondering if we should eventually make this more "dynamic"... not totally sure how we would but might be nice.